front/pages/w/[wId]/a/[aId]/datasets/index.tsx

```diff
@@ -122,9 +122,9 @@ export default function DatasetsView({
                               <div className="ml-2 flex flex-shrink-0">
                                 <TrashIcon
                                   className="hidden h-4 w-4 text-gray-400 hover:text-red-700 group-hover:block"
-                                  onClick={(e) => {
+                                  onClick={async (e) => {
                                     e.preventDefault();
-                                    handleDelete(d.name);
+                                    await handleDelete(d.name);
```

Comment:
I believe this change is a no-op. React is not going to await the promise returned by `onClick`, so if `handleDelete` throws, it's going to be an unhandled promise rejection. Maybe we should `.catch(console.error)` instead ?

Reply:
do we want to move this one to `void` @lasryaric  ?